spec/built_in_modules/color.md

```diff
@@ -0,0 +1,314 @@
+# Color Module
```

Comment:
Would it make sense to add TODOs or some other indicator for each function (here and in the other modules) whose semantics aren't formally specified yet?

Reply:
I think doing that in general would produce more noise than it's worth, since *most things* in the spec currently are unspecified. Maybe once it gets more fleshed-out?